packages/core/src/Script.ts

```diff
@@ -174,6 +194,10 @@ export class Script extends Component {
     if (this._onLateUpdateIndex !== -1) {
       componentsManager.removeOnLateUpdateScript(this);
     }
+    if (this._listenInput) {
```

Comment:
Do not add `if(this._listenInput)`， engine will ensure that the function is not called repeatedly.

Reply:
This is necessary, otherwise it is necessary to determine whether the method has been overwritten to determine the current listening state.